app/models/audit_results.rb

```diff
@@ -1,17 +1,6 @@
 # frozen_string_literal: true
 
-# AuditResults allows the correct granularity of information to be reported in various contexts.
-#   By collecting all the result information in this class, we keep the audit check code cleaner, and
-#   enable an easy way to provide:
-#    - the correct HTTP response code for ReST calls routed via controller
-#    - error information reported to workflows
-#    - information to Rails log
-#    - in the future, this may also include reporting to the provenance database as well
-#
-# All results are kept in the result_array attribute, which is returned by the report_results method.
-#   result_array = [result1, result2]
-#   result1 = {response_code => msg}
-#   result2 = {response_code => msg}
+# AuditResults allows the correct granularity of auditing information to be modeled in various contexts.
 class AuditResults
   ACTUAL_VERS_GT_DB_OBJ = :actual_vers_gt_db_obj
   ACTUAL_VERS_LT_DB_OBJ = :actual_vers_lt_db_obj
```

Comment:
Is `app/models/` the best place for this new version of the class? I don't know. But I think it's a better fit than `app/services/` and I'm not sure inventing a new `app/` subdirectory is better than stuffing this PORO into `app/models/` so :shipit: ?

Reply:
I think this is how stuff ended up in `app/lib` -- because it wasn't a rails model, which is all I expect to find in the `app/models` directory.   I'm not sure what's best;  Putting it in the services junk drawer feels more comfortable to me.  :shrug:   I would defer to John.

Reply:
@ndushay That'd work for me too. 😄

Reply:
In our other apps, we have non-Rails models in `app/models/`. At least based on my familiarity with our other apps, that is where I would have looked for it.

Reply:
I think you've convinced me `app/models/` is fine here.